cli/src/commands/balance/index.ts

```diff
@@ -288,26 +283,23 @@ class BalanceCommand extends Command {
     });
   }
 
-  private logTokenUtxos(balance: Balance, token: PublicKey, verbose = false) {
+  private logTokenUtxos(balance: Balance, token: PublicKey, _verbose = false) {
```

Comment:
Why "verbose" is prefixed with underscore?
btw, what do you think about it https://github.com/google/gts + https://google.github.io/styleguide/tsguide.html?

Reply:
prefixed because unused ; will add todo to add verbose utility back

Reply:
done